gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
I believe I have the solution I was looking for to remove the gulp and still get the debug ability.

https://gist.github.com/paambaati/54d33e409b4f7cf059cc

Up to date even!